### DISCUSSION TITLE
wifi image streamer opencv-viewer viewer not opening despite socket connected

### DISCUSSION THREAD

**Discussion by WolfscorpioTest** [bitcraze/discussions#1495] [Q&A]
I am trying to run the wifi image streamer example right now but it isn't working.

I used the JTAG cable to load the wifi image streamer, then I boatload the default firmware with crazy radio. I am using the command python3 opencv-viewer.py -n ipfromcfclient --save

Previously i used the crazyflie as its own access point which usually worked and opened the viewer but now it is always stuck on socket connected. In cfclient it says wifi: connection accepted and wifi: client connected but then the viewer never opens and images don't save.

Is there a way to debug this?

I added print statements to try and see where its freezing and its freezing right after packetInfoRaw = rx_bytes(4) but i don't know how to fix that.

**Comment by knmcguire** [comment]
Hi! Have you tried to make the aideck connect to an access point instead? Sometimes we observe some better streaming stability but it is difficult to figure out exactly why.

Here are also some instructions: https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/wifi-streamer/

**Reply by WolfscorpioTest** [reply]
Yea sorry I meant that I was trying to use it to connect to an access point when I was previously using it as its own. I ended up restarting and bootloading the crazyflie around three times and it seems to be working again now. Thank you.
